packages/arb-token-bridge-ui/src/components/TransferPanel/TokenButton.tsx

```diff
@@ -25,8 +24,7 @@ export function TokenButton(): JSX.Element {
   } = useAppState()
   const { status, l1, l2 } = useNetworksAndSigners()
 
-  const [tokenToImport, setTokenToImport] = useState<string>()
-  const [tokenImportDialogProps, openTokenImportDialog] = useDialog()
+  const { setTokenQueryParam } = useTokenFromSearchParams()
```

Comment:
This hook is not used in this file directly, we only pass it to `TokenSearch`, we should call it directly in `TokenSearch`